src/screens/resources/blogs-desktop.js

```diff
@@ -0,0 +1,72 @@
+import { VerticalScrollbar } from "../videos/vertical-scrollbar";
+import { blogs } from "./resources";
+import { cx } from "../../utils/cx";
+
+export function BlogsDesktop({ className }) {
+  return (
+    <div className={cx("grid grow grid-cols-12 gap-12", className)}>
+      <div className="col-span-7 flex flex-col">
+        <div className="flex grow flex-col">
+          <img
+            className="h-full rounded-t-xl object-cover"
+            src={`/images/other-resources/blogs/${blogs[0].key}.jpg`}
+            alt=""
```

Comment:
I believe it's technically mandatory:

From the MDN:

> The alt attribute is officially mandatory; it's meant to always be specified. If the image doesn't require a fallback (such as for an image which is decorative or an advisory icon of minimal importance), you may specify an empty string (""). For compatibility reasons, browsers generally will accept an image without an alt attribute, but you should try to get into the habit of using it.

Reply:
Oh I meant fill in a real alt like `alt=someText`, or is there a reason we don't want an alt?

Reply:
The way I see it, `alt` should be used to provide semantic value. Since the images are only decorative in this case, we don't really need to have a real value.

Reply:
makes sense 👍